overlay/addons/external-db-no-tls.yml

```diff
@@ -1,12 +1,19 @@
 # Requires the external-db to be enabled
 ---
+meta:
+  registry_db:
+    tls:
+      enabled: false
+
 instance_groups:
   - name: bosh
     properties:
       director:
         db:
           tls:
             enabled: false
+      registry:
```

Comment:
This isn't needed in this file, the one in external-db.yml will suffice

Reply:
removed